python/utils.py

```diff
@@ -677,6 +677,13 @@ def fcnRegexp(pattern, string):
             break
     if not found:
         raise RuntimeError("Cannot find any suitable spatialite module")
+    if any(['.gpkg' in arg for arg in args]):
+        try:
+            cur.execute("SELECT EnableGpkgAmphibiousMode()")
+        except:
```

Comment:
Final thing, can you catch specific exceptions here? Catch all exceptions tend to give misleading error messages.